annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -719,10 +719,12 @@ private TranscriptConsequence transcriptWithMostSevereConsequence(List<Transcrip
     }
 
     private String getVariantClassificationFromMap(String variant) {
-        boolean inframe = Math.abs(mRecord.getREFERENCE_ALLELE().length() - getTumorSeqAllele(mRecord).length()) % 3 == 0;
+        String refAllele = mRecord.getREFERENCE_ALLELE().equals("-") ? "" : mRecord.getREFERENCE_ALLELE();
```

Comment:
So on an insert the reference_allele field is exactly one hyphen ; and on a delete the tumor_seq_allele is exactly one hyphen. This extra hyphen (when present) must be stripped before doing the modulo 3 math.